src/Store/InputStoreConnector.re

```diff
@@ -58,6 +58,10 @@ let conditionsOfState = (state: State.t) => {
     Hashtbl.add(ret, "editorTextFocus", true);
   | (false, Vim.Types.Visual) => Hashtbl.add(ret, "visualMode", true)
   | (false, _) => Hashtbl.add(ret, "editorTextFocus", true)
+  | (true, Vim.Types.CommandLine) =>
+    // LEGACY: Support `commandLineFocus` too.
+    Hashtbl.add(ret, "commandLineFocus", true);
+    Hashtbl.add(ret, "commandLineMode", true);
```

Comment:
Why add `commandLineMode` if it's exactly the same as the existing `commandLineFocus`?

Reply:
To be consistent with the other modes - we'll eventually have:
- `insertMode`
- `normalMode`
- `visualMode`
...

It seems inconsistent... although I guess it's debatable if `commandLine` _should_ be a mode. If we decide to build a 'first-class' command-line mode (like, powered by our text input model + menu infra, with completion, etc), I see that as a good argument for `commandLineFocus` instead though. I'll switch it back.

Reply:
Ah, I see. Yeah, from a normal user POV `commandlineFocus` makes much more sense I think, but from a vim POV `commandlineMode` definitely makes more sense. I'd lean towards favoring the user POV too though.